pkg/controller/xqueuejob/queuejob_controller_ex.go

```diff
@@ -0,0 +1,240 @@
+/*
```

Comment:
please remove the controller in this pr.

Reply:
@k82cn : see PR#253

Reply:
Would you sync up before open PRs ?